daemon/daemon_unix.go

```diff
@@ -1190,7 +1190,9 @@ func (daemon *Daemon) initCgroupsPath(path string) error {
 		return nil
 	}
 
-	daemon.initCgroupsPath(filepath.Dir(path))
```

Comment:
I think the recursion is needed because all of the parents have to have a value `<=` than the targeted one in the path for this to work if I'm not mistaken